integration/spark2/src/main/scala/org/apache/spark/sql/hive/execution/command/CarbonHiveCommands.scala

```diff
@@ -95,8 +95,14 @@ object CarbonSetCommand {
       }
     } else if (key.startsWith(CarbonCommonConstants.VALIDATE_CARBON_INPUT_SEGMENTS)) {
       sessionParams.addProperty(key.toLowerCase(), value)
+    } else if (key.startsWith(CarbonCommonConstants.IS_INTERNAL_LOAD_CALL)) {
```

Comment:
I don't think it is required to use `set command` for this internal call. we are not going to give the option to load the aggregate table as it may corrupt the table.